web/src/components/FOI/Home/Home.js

```diff
@@ -52,14 +52,31 @@ function Home() {
     fetchDocuments(
       parseInt(foiministryrequestid),
       async (data) => {
+
+        const getFileExt = (filepath) => {
+          const parts = filepath.split(".")
+          const fileExt = parts.pop()
+          return fileExt
+        }
         // New code added to get the incompatable files for download redline
         // data has all the files including incompatable ones
         // _files has all files except incompatable ones
         const _incompatableFiles = data.filter(
```

Comment:
Not sure what these incompatible files are being used for, but they're being passed to Redlining component. This code filters by filepath extension, so before replacing the incompatible file will be added to _incompatablefiles and not in _files, after replacing it will be in _files and not in _incompatablefiles